### PR TITLE
V0.13.0

### PR DESCRIPTION
**Pull request by meetar**

### CODE REVIEW

File: pages/Javascript-API.md [hunk 1]
```diff
@@ -60,25 +62,86 @@ The promise resolves with a `selection` object:
...
+   ...
+});
+```
+
 #### `loadTextures()`
 Reloads and rebinds [textures](textures.md) in the scene.
 
 Textures will only be reloaded if any of the texture definition's parameters changed, or if the texture is tied to a DOM element.
 
+#### `queryFeatures()`
```
Comment (bcamper) [inline pages/Javascript-API.md:121]: There are several other options we need to incorporate here, such as `visible`, `unique`, `group_by`, `geometry`. These should be well documented (along with some examples that I think are worth incorporating) in https://github.com/tangrams/tangram/pull/566.

**Review by bcamper — CHANGES_REQUESTED** [review]
Made some small changes, but want to make sure we incorporate full docs for `queryFeatures()` method too.

**Comment by bcamper** [comment]
@meetar -- we still need to incorporate the `queryFeatures()` details.

**Comment by bcamper** [comment]
Good to merge?

**Comment by meetar** [comment]
:+1:

**Pull request merged**